apps/minifront/src/state/swap/index.ts

```diff
@@ -18,58 +18,10 @@ import {
 } from './instant-swap';
 import { createPriceHistorySlice, PriceHistorySlice } from './price-history';
 import { getMetadata } from '@penumbra-zone/getters/value-view';
-import { createZQuery, ZQueryState } from '@penumbra-zone/zquery';
-import { getSwappableBalancesResponses, isSwappable } from '../../components/swap/helpers';
-import { getAllAssets } from '../../fetchers/assets';
-import { emptyBalanceResponse } from '../../utils/empty-balance-response';
 import { isValidAmount } from '../helpers';
 
-// When both `balancesResponses` and `swappableAssets` are loaded, set initial assetIn and assetOut
-const setInitialAssets = (state: SwapSlice) => {
```

Comment:
The `swap` slice no longer has its own `balancesResponses` and `swappableAssets` ZQuery objects, since those have been moved to the `shared` slice. We don't want the `shared` slice to have to worry about setting the asset in/out for the `swap` slice, since that would create a circular dependency of sorts between the `shared` and `swap` slices. Instead, the functionality of `setInitialAssets` has been moved to the middlewares at the bottom of this file.